plugins/file/index.js

```diff
@@ -87,7 +87,15 @@ module.exports = {
           return Boom.unsupportedMediaType("Content-type not allowed");
         }
 
-        const type = mimos.type(contentType);
+        // The mimos package doesn't store the extension for the content-type text/javascript
+        // because its marked as depricated by IANA. Therefore we get the file extension based
```

Comment:
just a small type: `deprecated`

Reply:
Fixed in https://github.com/nzzdev/Q-server/pull/137/commits/1e2eb46d68536addfb1cd3f22cecd5e54d631c7c